app/models/post.rb

```diff
@@ -1,3 +1,7 @@
 class Post < ApplicationRecord
-  belongs_to :author_id
+  belongs_to :user
+
+  scope :last_user_posts, ->(user_id) { where(user_id:).last(3) }
+  scope :increace_comments_count, ->(post_id) { find_by(id: post_id).increment!(:comments_counter) }
+  scope :increace_likes_count, ->(post_id) { find_by(id: post_id).increment!(:likes_counter) }
```

Comment:
- [ ] I think this method it should be in likes model so kindly add it there according to the requirements so please update your comments model to have it there 💯🙏🏻🔥

![image](https://user-images.githubusercontent.com/58553711/220621447-18a0b233-17bc-4b30-81fb-f2f8b19fa81c.png)



-  Hint: I would highly recommend to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new like has been saved to database, then make the `update_likes_counter` method private. 🙏🏻💯